core/modules/czar/Czar.cc

```diff
@@ -68,8 +80,15 @@ Czar::Czar(std::string const& configPath, std::string const& czarName)
         LOG_CONFIG(logConfig);
     }
 
-    int largeResultPoolSize = _czarConfig.getLargeResultPoolSize();
-    rproc::InfileMerger::setLargeResultPoolSize(largeResultPoolSize);
+    int largeResultConcurrent = _czarConfig.getLargeResultConcurrentMerges();
+    LOGS(_log, LOG_LVL_INFO, "config largeResultConcurrent=" << largeResultConcurrent);
+    _largeResultMgr = std::make_shared<qdisp::LargeResultMgr>(largeResultConcurrent);
+
+    int xrootdCBThreadsMax = _czarConfig.getXrootdCBThreadsMax();
+    int xrootdCBThreadsInit = _czarConfig.getXrootdCBThreadsInit();
+    LOGS(_log, LOG_LVL_INFO, "config xrootdCBThreadsMax=" << xrootdCBThreadsMax);
+    LOGS(_log, LOG_LVL_INFO, "config xrootdCBThreadsInit=" << xrootdCBThreadsInit);
+    XrdSsiProviderClient->SetCBThreads(xrootdCBThreadsMax, xrootdCBThreadsInit);
```

Comment:
Well, xrootdCBThreadsInit is really not descriptive of what that parameter does. The first argument is the maximum number of threads to use for callbacks. The second parameters is the number of threads to set aside to handle network events. Just my thing here.